frontend/src/components/pod/List.tsx

```diff
@@ -127,7 +127,7 @@ interface PodTableRow {
   lastRestartDate: Date;
 }
 
-// Source Code: https://github.com/kubernetes/kubernetes/blob/33bf8a8ebd33eae0d45c18feb1c05baa6a055cde/pkg/printers/internalversion/printers.go#L558
+// Source Code: https://github.com/kubernetes/kubernetes/blob/master/pkg/printers/internalversion/printers.go#L759
```

Comment:
I know @illume asked for this, but I am not sure if having a link to a branch is better than to the actual commit, since the code will likely change.